docs/source/algorithms/Abins-v1.rst

```diff
@@ -39,6 +39,8 @@ Abins is in constant development and suggestions
 how to make it better are very welcome. For any suggestions of enhancements please contact
 Krzysztof Dymkowski (krzysztof.dymkowski@stfc.ac.uk).
 
+If abINS is used as part of your data analysis routines, please cite the relevant reference [1]_.
```

Comment:
The rest of the document uses `Abins` (algorithm name) and not `abINS`. I think here we are referring to the algorithm so let's use that.